src/cql/r4/Factors_to_Consider_in_Managing_Chronic_Pain_FHIRv400.cql

```diff
@@ -917,6 +924,37 @@ define ReportUrineDrugScreens:
   sort by Date desc
 
 
+// Urine drug screen from service request,  matching on LOINC code and system = http://loinc.org
+define ReportUrineLabServiceRequests:
+  ([ServiceRequest: "Urine drug screen for pain management"]) O
+  let c: O.code as FHIR.CodeableConcept
+  return {
+    Name: ConceptText(c),
+    Date: DateTimeText(O.authoredOn)
+  }
+  sort by Date desc
+
+// Urine drug screen from service request,  matching on custom system and concept code, e.g. those from Skagit site
```

Comment:
It's true this change is for Skagit, but it's potentially broader.

Probably more accurate to change these comments to "sites using labcorp for testing"

Reply:
thank you, fixed